src/sagemaker_inference/model_server.py

```diff
@@ -86,13 +96,19 @@ def _adapt_to_mms_format(handler_service):
     if not os.path.exists(DEFAULT_MMS_MODEL_DIRECTORY):
         os.makedirs(DEFAULT_MMS_MODEL_DIRECTORY)
 
-    model_archiver_cmd = ['model-archiver',
-                          '--model-name', DEFAULT_MMS_MODEL_NAME,
-                          '--handler', handler_service,
-                          '--model-path', environment.model_dir,
-                          '--export-path', DEFAULT_MMS_MODEL_DIRECTORY,
-                          '--archive-format', 'no-archive',
-                          ]
+    model_archiver_cmd = [
```

Comment:
Is this (and the immediate above) enforced by black-format? I feel like the old format makes more sense in the context, but if it is enforced, let's keep them this way.

Reply:
I agree with you; the way it was formatted before is visually cleaner. This was enforced by `black-format`, though, so we can keep the change.